.github/ISSUE_TEMPLATE/feature-request-template.yml

```diff
@@ -0,0 +1,65 @@
+name: Feature Request
+description: File a feature request issue
+title: "[Feature Request]: <Please Replace This with a Summary of Your Request>"
+labels: ["addition", "p2", "needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you so much for taking the time to fill out this feature request report! Please note that changing the issue label from p2 to p0/p1 will *NOT* speed up triaging. **Any issue that is currently affecting your production should be directed to Google Cloud customer support.** If you cannot find the template in the dropdown menu below, we probably no longer support the template.
```

Comment:
I think the following is good enough:

```
Thank you so much for taking the time to fill out this feature request!
```

The rest seems more relevant to bug reports.